mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/Util.java

```diff
@@ -32,7 +32,7 @@ public static double getEffectiveValue(StageSchedulingInfo stageSchedulingInfo,
         case Memory:
```

Comment:
Should we annotate ScalingReason.Memory as deprecated?

Reply:
done